objects/SetupChecker/ui.xml

```diff
@@ -172,7 +173,7 @@
         </TableLayout>
     </VerticalLayout>
 </Panel>
-<Panel id="panelVariant" recurse="events" visibility="Invisible" width="1000" height="2800" position="-2000 -3160 14">
+<Panel id="panelVariant" recurse="events" visibility="Invisible" width="1000" height="1000" position="-2500 -1760 14" outline="#FFFFFF">
```

Comment:
why is outline getting added here?

Reply:
Removed. That was one attempt to visualize the actual size of the panels that didn't work. I guess I missed deleting it when preparing the final patch.